templates/components/global-footer/global-footer.html.twig

```diff
@@ -4,6 +4,38 @@
  * Default theme implementation for the basic structure of the global stanford footer
  */
 #}
+
+<section class="contact-footer" style="background-image:url('https://www-tc.pbs.org/wnet/nature/files/2017/03/1007.jpg')";>
+  <div class="contact-footer__brand">
+    <dd>{{ drupal_block('matson_branding') }}</dd>
+  </div>
+  <div class="contact-footer__social">
+    <h2>Join us on Social Media</h2>
```

Comment:
Please translate your strings. See: https://drupal.stackexchange.com/questions/192498/how-do-i-translate-strings-in-twig-templates